File: server/Models/Comment.cs
namespace theJourney.Models;
public class Comment
{
    public int Id { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public string Body { get; set; }
    public int PostId { get; set; }
    public Account Creator { get; set; }
    public string CreatorId { get; set; }
}

File: client/src/models/Comment.js
export class Comment{
    constructor(data){
        this.id = data.id
        this.body = data.body
        this.postId = data.postId
        this.creator = data.creator
        this.creatorId = data.creatorId
        this.createdAt = new Date(data.createdAt).toLocaleTimeString()
        this.updatedAt = new Date(data.updatedAt).toLocaleDateString()
    }
}